src/main/java/org/generationcp/middleware/hibernate/DataSourceProperties.java

```diff
@@ -92,6 +92,14 @@ public DataSourceProperties(final Properties properties) {
 
 	}
 
+	private String getPropertyValue(final String envVariable, final Properties properties, final String defaultValue, final String property) {
+		final String valueInEnvironment = System.getenv(envVariable);
+		if (Strings.isNullOrEmpty(valueInEnvironment)) {
```

Comment:
Can be simplified as:
`return Strings.isNullOrEmpty(valueInEnvironment)? getPropertyValue(properties, defaultValue, property) : valueInEnvironment;`